### ISSUE TITLE
PixelFormats.Rgba8888 not working on Linux

### ISSUE THREAD

**Issue by arklumpus** [AvaloniaUI/Avalonia#4354]
Hello, I noticed that if I create a WriteableBitmap with PixelFormats.Rgba8888 and then "manually" set the pixel data, the image is displayed on Linux as if it had been created with PixelFormats.Bgra8888 (i.e. the red and blue channel are switched). It looks as expected on Windows:

![LinuxColour](https://user-images.githubusercontent.com/32712606/88313643-1c88f880-cd14-11ea-888d-09cc136f9fc0.png)

Changing to PixelFormats.Bgra8888, the colours on Windows change (as expected), while the colours on Linux stay the same (which are correct, in this case).

To reproduce this, create a new Avalonia application and replace the constructor of the MainWindow with the following:

```csharp
WriteableBitmap bmp = new WriteableBitmap(new PixelSize(100, 100), new Vector(96, 96), PixelFormat.Rgba8888);

            using (ILockedFramebuffer fb = bmp.Lock())
            {
                int stride = 100 * 4;

                unsafe
                {
                    byte* data = (byte*)fb.Address;

                    for (int y = 0; y < 50; y++)
                    {
                        //Red
                        for (int x = 0; x < 50; x++)
                        {
                            data[y * stride + x * 4] = 255;
                            data[y * stride + x * 4 + 1] = 0;
                            data[y * stride + x * 4 + 2] = 0;
                            data[y * stride + x * 4 + 3] = 255;
                        }

                        //Green
                        for (int x = 50; x < 100; x++)
                        {
                            data[y * stride + x * 4] = 0;
                            data[y * stride + x * 4 + 1] = 255;
                            data[y * stride + x * 4 + 2] = 0;
                            data[y * stride + x * 4 + 3] = 255;
                        }
                    }

                    for (int y = 50; y < 100; y++)
                    {
                        //Blue
                        for (int x = 0; x < 50; x++)
                        {
                            data[y * stride + x * 4] = 0;
                            data[y * stride + x * 4 + 1] = 0;
                            data[y * stride + x * 4 + 2] = 255;
                            data[y * stride + x * 4 + 3] = 255;
                        }

                        //Grey
                        for (int x = 50; x < 100; x++)
                        {
                            data[y * stride + x * 4] = 128;
                            data[y * stride + x * 4 + 1] = 128;
                            data[y * stride + x * 4 + 2] = 128;
                            data[y * stride + x * 4 + 3] = 255;
                        }
                    }
                }
            }

            this.Content = new Image() { Source = bmp };
```

I tested this on Ubuntu 20.04 in a virtual machine. It happens both with Avalonia 0.9.11 and 0.10.0-preview1.

**Comment by kekekeks** [comment]
I wonder why do we have this line: https://github.com/AvaloniaUI/Avalonia/blob/e65203be4e92e418f236c54992a29ca36880be5e/src/Skia/Avalonia.Skia/Helpers/PixelFormatHelper.cs#L24

**Comment by arklumpus** [comment]
This appears to be fixed in 0.10.0-rc1
